aloe/result.py

```diff
@@ -38,36 +38,60 @@
 # started, which is when the stream is passed in.
 TERMINAL = [None]
 
-try:
+try:    # pylint: disable=too-complex
     from blessings import Terminal as BaseTerminal
 except ImportError:
-    # When curses is unavailable, fall back to not coloring the output.
+    import colorama
+    from colorama import Fore, Cursor
+    from colorama.ansi import CSI
 
-    class NullCallableString(str):  # pragma: no cover
-        """Empty string that returns its argument when called."""
+    colorama.init()
 
-        def __new__(cls):
-            new = str.__new__(cls, '')
-            return new
+    class FormattingString(str):
+        """Alternative non-curses/blessings FormattingString implementation."""
 
-        def __call__(self, *args):
-            return args[0]
+        def __new__(cls, formatting):
```

Comment:
Is this `__new__` method needed? It doesn't modify the default behavior.

Reply:
Gone. Just an artefact from trying to replicate `blessings` as closely as I could at the start.